packages/monaco-editor-wrapper/src/monacoVscodeApiWrapper.ts

```diff
@@ -0,0 +1,44 @@
+import { updateUserConfiguration } from 'vscode/service-override/configuration';
+import { registerExtension, IExtensionManifest } from 'vscode/extensions';
+import { EditorConfig, MonacoEditorWrapper } from './wrapper.js';
+import 'vscode/default-extensions/theme-defaults';
+
+export type VscodeUserConfiguration = {
+    json?: string;
+}
+
+export type MonacoVscodeApiWrapperConfig = {
+    extension?: IExtensionManifest | object;
+    extensionFiles?: Map<string, URL>;
+    userConfiguration?: VscodeUserConfiguration;
+}
+
+export class MonacoVscodeApiWrapper implements MonacoEditorWrapper {
+
+    async init(_editorConfig: EditorConfig, wrapperConfig: MonacoVscodeApiWrapperConfig) {
+        console.log(window.location.href);
```

Comment:
Is this needed?

Reply:
No, thank you for spotting this. Left-over.

Reply:
Fixed.